backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -100,4 +102,18 @@ module.exports = class CrmPage {
       this.engine.waitForNavigation()
     ]);      
   }
+
+  /**
+   * Checks if the checkbox is enabled and checks it if not.
+   *
+   * @param {String} selector - the css selector for the checkbox to be click and enabled
+   */
+  async enableCheckbox (selector) {
+    const  checkbox = await this.engine.$(selector);
+    const checked = await (await checkbox.getProperty('checked')).jsonValue();
```

Comment:
does `jsonValue` needs an `await`?

Reply:
Yes, it does! It returns Promise Object - https://github.com/GoogleChrome/puppeteer/blob/v1.5.0/docs/api.md#jshandlejsonvalue